### DISCUSSION TITLE
album multiselection

### DISCUSSION THREAD

**Discussion by rix30** [immich-app/immich#11059] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi all, thx for this GREAT platform!
I 've loaded a bunch of album from google photos with immich_go, now I need to share the created albums with my wife, they're over 130, I din't find a way to multiselect (or select all ) and share them with my wife in one operation, now I must select and share one by one. I think it could be a great help for people like me.
Hi and great thanks, I love Immich!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#8543
